site/content/3.13/graphs/graph-visualizer/_index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Graph Visualizer
+menuTitle: Graph Visualizer
+weight: 85
+description: >-
+  Graph Visualizer allows you to visualize the graphs present in the database.
+---
+
+A **Graph** in ArangoDB is a powerful data structure that models relationships between entities. The **Graph Visualizer** provides an interactive interface to explore these connections, enabling users to traverse edges, inspect vertices, and intuitively understand the underlying data structure.
+
+> 💡 **Note:** Graph creation is **not** performed within the Graph Visualizer. Graphs must be created in the **Graphs** section of the application. Once a graph is created there, it will automatically appear in the Graph Visualizer, ready for exploration and visualization.
+
+## 1.Data Augmentation
+
+While the Graph Visualizer is primarily designed for exploring graph data, you can also create and modify nodes and edges directly from the canvas.
+
+**A.Add New Nodes (Vertices):** 
+  Right-click anywhere on the canvas and select **"Create Node"**. A dropdown will appear showing all related document collections. Select the desired collection (vertex type), provide a unique identifier or name, and create the node. The node will then appear in the visualization.
```

Comment:
- UI element labels should just be bold but not be wrapped in quote marks. Please see the reference page again: https://docs.arangodb.com/stable/components/web-interface/graphs/
- It's a dropdown **menu**. Maybe the term "context menu" is preferable? You can check the existing documentation, whether there is an established term already.
- Avoid using "will" etc., see https://developers.google.com/style/tense